linux_sql/README.md

```diff
@@ -0,0 +1,100 @@
+# Linux Cluster Monitoring Agent
+
+## Introduction
+  
+The Linux Cluster Administration (LCA) team manages a Linux cluster of 10 nodes/servers which are running CentOS 7. These servers are internally connected through a switch and able to communicate through internal IPv4 addresses. The goal of the monitoring agent is to record the hardware specifications of each node and monitor node resource usages (e.g. CPU/Memory) in realtime. The monitoring agent is implemented using bash scripts and the PostgreSQL database, and the results of the agent will then be stored in the RDBMS database.
```

Comment:
you pretty much copied the instruction from the scrum board. please be creative.